pkg/db/interface_test.go

```diff
@@ -119,7 +119,7 @@ func TestGetStudyConfig(t *testing.T) {
 	mock.ExpectQuery("SELECT").WillReturnRows(
 		sqlmock.NewRows(studyColumns).AddRow(
 			"abc", "test", "admin", 1, 0.99, "{}", "", "", "", "test"))
-	study, err := dbInterface.GetStudyConfig(id)
+	study, err := dbInterface.GetStudy(id)
 	if err != nil {
```

Comment:
You should also change this things: 
1. Add 2 values in StudyColumns: https://github.com/kubeflow/katib/blob/297324dd7d88c32d81c63b7109f051e558db8a44/pkg/db/interface_test.go#L23
2. Add value "time" in trialColumns:
https://github.com/kubeflow/katib/blob/297324dd7d88c32d81c63b7109f051e558db8a44/pkg/db/interface_test.go#L27
3. Add 2 empty values:
https://github.com/kubeflow/katib/blob/297324dd7d88c32d81c63b7109f051e558db8a44/pkg/db/interface_test.go#L119
4. Add 1 empty value:
https://github.com/kubeflow/katib/blob/297324dd7d88c32d81c63b7109f051e558db8a44/pkg/db/interface_test.go#L216
5. Add 2 Values empty values:
https://github.com/kubeflow/katib/blob/297324dd7d88c32d81c63b7109f051e558db8a44/pkg/db/interface_test.go#L367
6. Add 1 empty value:
https://github.com/kubeflow/katib/blob/297324dd7d88c32d81c63b7109f051e558db8a44/pkg/db/interface_test.go#L233